tests/FakeItEasy.Tests/Core/DefaultFakeManagerAccessorTests.cs

```diff
@@ -10,12 +10,7 @@
 
     public class DefaultFakeManagerAccessorTests
     {
-        private readonly DefaultFakeManagerAccessor accessor;
-
-        public DefaultFakeManagerAccessorTests()
-        {
-            this.accessor = new DefaultFakeManagerAccessor();
-        }
+        private readonly IFakeManagerAccessor accessor = ServiceLocator.Current.Resolve<IFakeManagerAccessor>();

```

Comment:
Are you sure about this change? This test fixture is `DefaultFakeManagerAccessorTests`, but it never explicitly creates a `DefaultFakeManagerAccessor`; instead it relies on the service locator to return a `DefaultFakeManagerAccessor`. This seems wrong...

Reply:
I am not sure at all. I have no idea why I did that. (Nor any memory of it, but that's not unusual.) Thank you!